README.md

```diff
@@ -148,6 +148,14 @@ class Account < ApplicationRecord
 end
 ```
 
+Custom index can be handled with a `Arel::Nodes::SqlLiteral` instance:
```

Comment:
I'm ready to merge this, and polish the wording of this documentation note later!

Reply:
Oups, my bad